__snapshots__/make-badge.spec.js

```diff
@@ -1,135 +1,1947 @@
 exports['The badge generator SVG should match snapshot 1'] = `
```

Comment:
I gotta be honest, can't see myself doing a detailed line-by-line review of this file :smile: I assume there's no additional delta beyond the prettification and that our tests give us confidence that prettier didn't somehow botch them?

Reply:
Oh, yea, you mean in this PR. 😁 Yea, the changes to that file were all auto-generated.